datacite-delete-draft-doi-handler/src/main/java/no/unit/nva/datacite/handlers/resource/ResourceDraftedForDeletionEventHandler.java

```diff
@@ -100,6 +101,7 @@ private ResourceDraftedForDeletionEvent deleteDraftPublication(
     }
 
     private ResourceDraftedForDeletionEvent copyDeletePublicationWithoutDoi(ResourceDraftedForDeletionEvent event) {
+        //Is this event necessary?
```

Comment:
La inn kommentar i koden så jeg kunne få med det her. Jeg er usikker på om denne trenger å sende noe beskjed videre? @ketilaa vet du?

Reply:
Lar det stå for nå. Da jeg ikke er 100% sikker.